bip-0015.mediawiki

```diff
@@ -36,7 +36,7 @@ Their FirstBits alias becomes:
 
 It is enough information to be given the FirstBits alias ''1brmlab''. When someone wishes to make a purchase, without FirstBits, they either have to type out their address laboriously by hand, scan their QR code (which requires a mobile handset that this author does not own) or find their address on the internet to copy and paste into the client to send bitcoins. FirstBits alleviates this impracticality by providing an easy method to make payments.
 
-Together with [[vanitygen|Vanitygen (vanity generator)]], it becomes possible to create memorable unique named addresses. Addresses that are meaningful, rather than an odd assemblage of letters and numbers but add context to the destination.
+Together with [vanitygen https://github.com/klynastor/supervanitygen], it becomes possible to create memorable unique named addresses. Addresses that are meaningful, rather than an odd assemblage of letters and numbers but add context to the destination.
```

Comment:
As far as I can tell from the commit history of this repository, this link was always broken. Rather than assigning it a destination that may or may not match the BIP Champion’s intention, it would be preferable to remove the link altogether.

Reply:
Good point.

I have emailed the BIP Champion for his input and asked him to review this PR.

Reply:
> it would be preferable to remove the link altogether.

Good point.
I have emailed the BIP Champion for his input and asked him to review this PR.

Reply:
> I have emailed the BIP Champion for his input and asked him to review this PR.

The email bounced.  Now what?

![image](https://github.com/bitcoin/bips/assets/967793/8597ecbc-4ff2-4045-b788-51f17bcd315a)